### ISSUE TITLE
Gulp-cli v 2.2.0 fails if task export using parallel or series functions

### ISSUE THREAD

**Issue by rikani** [gulpjs/gulp-cli#193]
After update to v2.2.0 gulp fails with error: `Task never defined: default`

For example I tried the code from https://gulpjs.com Home page: 
```
const { src, dest, parallel } = require('gulp');
const pug = require('gulp-pug');
const less = require('gulp-less');
const minifyCSS = require('gulp-csso');
const concat = require('gulp-concat');

function html() {
  return src('client/templates/*.pug')
    .pipe(pug())
    .pipe(dest('build/html'))
}

function css() {
  return src('client/templates/*.less')
    .pipe(less())
    .pipe(minifyCSS())
    .pipe(dest('build/css'))
}

function js() {
  return src('client/javascript/*.js', { sourcemaps: true })
    .pipe(concat('app.min.js'))
    .pipe(dest('build/js', { sourcemaps: true }))
}

exports.js = js;
exports.css = css;
exports.html = html;
exports.default = parallel(html, css, js);
```
Command `gulp --tasks` shows next list of tasks:
```
 ├── js
 ├── css
 ├── html
 └─┬ <parallel>
   └─┬ <parallel>
     ├── html
     ├── css
     └── js
```
There is no `default` task.

Gulp-cli v 2.2.0 is not compatible with the `gulpfile.js` of the previous version.

**Comment by rikani** [comment]
https://github.com/gulpjs/gulp-cli/issues/191

**Comment by phated** [comment]
@rikani I've also shipped v4.0.1 to help ease the lockfile issues with that.  And since, 4.0.2 has been released that fixed another bug so those 2 versions should be used over 4.0.0 + updated undertaker.
